index.html

```diff
@@ -1292,20 +1295,24 @@ <h4>Base Proof Serialization (bbs-2023)</h4>
                 </li>
                 <li>
 If |commitment_with_proof| is not empty and |use_pseudonyms| is true, compute
-the `bbsSignature` using the `Sign` procedure of [[CFRG-Pseudonym-BBS-Signature]],
-with appropriate key material, `bbsHeader` for the `header`, `bbsMessages`
-for the `messages`, and |commitment_with_proof| for the `commitment_with_proof`.
-This provides for "pseudonym with hidden pid".
+the `bbsSignature` using the `Sign` procedure of [[CFRG-Blind-BBS-Signature]]
+and [[CFRG-Pseudonym-BBS-Signature]], with appropriate key material, `bbsHeader`
+for the `header`, `bbsMessages` for the `messages`, and |commitment_with_proof|
+for the `commitment_with_proof`. If the signing procedure uses the optional
+|signer_blind| parameter retain this value for use when calling
+<a href="#serializebaseproofvalue"></a> below.
+This provides for the "pseudonym with hidden pid" feature.
```

Comment:
_**This must be answered before the suggestion can be complete:**_ Is it ```using the `Sign` procedure of``` *both* ```[[CFRG-Blind-BBS-Signature]] and [[CFRG-Pseudonym-BBS-Signature]]```? Or is it ```using the `Sign` procedure of``` *whichever is relevant, Blind or Pseudonym* ```[[CFRG-Blind-BBS-Signature]]``` _or_ ```[[CFRG-Pseudonym-BBS-Signature]]```?
```suggestion
the `bbsSignature` using the `Sign` procedure of [[CFRG-Blind-BBS-Signature]]
and [[CFRG-Pseudonym-BBS-Signature]], with appropriate key material, `bbsHeader`
for `header`, `bbsMessages` for `messages`, and |commitment_with_proof|
for `commitment_with_proof`. If the signing procedure uses the optional
|signer_blind| parameter, retain this value for use when calling
<a href="#serializebaseproofvalue"></a> (below).
This provides for the "pseudonym with hidden pid" feature.
```

Reply:
I don't know yet. I should have a better idea after next Mondays DIF BBS call. I'd find it easiest if we roll the pseudonym work into the Blind Signature draft which is more mature. Note that the cryptography is very mature.

Reply:
+1 to just go with what's there today (use pseudonym work).